//Define the used version of proto
syntax = "proto3";

// This data type includes all the fields for microsensor and meteorological data. Note: is synchronized with the same message type in the calibration service.
message DataSample {
  float pm1 = 1;
  float pm25 = 2;
  float pm10 = 3;
  float air_temperature = 4;
  float relative_humidity = 5;
  float precipitation = 6;
  float air_pressure = 7;
  float wind_speed = 8;
  float wind_direction = 9;
}

message Empty {
}

service AQDataSource {
  rpc request_update(Empty) returns (DataSample);
}
